tests/ha/ha_install.pm

```diff
@@ -0,0 +1,46 @@
+# SUSE's SLES4SAP openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Install HA
+# Maintainer: QE-SAP <qe-sap@suse.de>
+
+use base 'sles4sap';
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+use power_action_utils 'power_action';
+use strict;
+use warnings;
+
+sub run {
+    my ($self) = @_;
+
+    select_console 'root-console';
+
+    # Workaround for current agama installation: registration, extensions and so on
+    # Install suseconnect-ng
+    zypper_call('in suseconnect-ng');
```

Comment:
It should be installed by default

Reply:
For sles4sap it is installed by default but for sles command `suseconnect` is not found.